default.py

```diff
@@ -28,10 +29,14 @@
 cookie_file = os.path.join(ADDON_PROFILE, 'cookie_file')
 username = addon.getSetting('email')
 password = addon.getSetting('password')
-if addon.getSetting('debug') == 'false':
-    debug = False
-else:
-    debug = True
+debug_cmd = {  # determine if debug logging is activated in kodi
+    'jsonrpc': '2.0',
```

Comment:
It looks like it, unfortunately. :/ I tried to grab it using both xbmc.GetSetting and xbmc.getCondVisibility, but this was the only way I could get it to work.

We might be able to use it for all settings, but I'm really not sure... I'm finding the documentation of the API quite difficult to understand, but maybe you can get a better hold of it. http://kodi.wiki/view/JSON-RPC_API/v6